### PR TITLE
Bug 1817562: Configure tls for multus metrics endpoint

### PR DESCRIPTION
**Pull request by aneeshkp**
Securing multus scrape endpoint with kube-rbac-proxy

### CODE REVIEW

File: manifests/0000_70_cluster-network-operator_03_deployment.yaml [hunk 1]
```diff
@@ -31,6 +31,8 @@ spec:
           value: "quay.io/openshift/origin-sdn:4.3"
         - name: KUBE_PROXY_IMAGE
           value: "quay.io/openshift/origin-kube-proxy:4.3"
+        - name: KUBE_RBAC_PROXY_IMAGE
+          value: "quay.io/openshift/origin-kube-rbac-proxy:latest"
```
Comment (lilic) [inline manifests/0000_70_cluster-network-operator_03_deployment.yaml]: Nit: Any specific reason this is latest and the other are pinned tags?

Comment (aneeshkp) [reply manifests/0000_70_cluster-network-operator_03_deployment.yaml]: No. I am not sure how it works ,  these get replaced by release manager I guess, typically like an art.yaml file that ties to image-references.yaml which will get replaced by an image name that the build system understands. Something like that.

Comment (aneeshkp) [reply manifests/0000_70_cluster-network-operator_03_deployment.yaml]: @juanluisvaladas I update the image tag  from latest to 4.3

**Comment by aneeshkp** [comment]
@juanluisvaladas I updated the tag to 4.3

**Comment by aneeshkp** [comment]
> What address is multus listening on? I see the kube-rbac-proxy points to 127.0.0.1:9091 but I don't see any change in multus config to listen on 127.0.0.1:9091 instead of 0.0.0.0:9091
@juanluisvaladas, I didn't change because 0.0.0.0 will also listen to 127.0.0.1, Do you think I should change bind-address to 127.0.0.1 only?

**Comment by juanluisvaladas** [comment]
/lgtm
@aneeshkp thinking about it, you're right about listening on 0.0.0.0. It's probably innecessary to limit the listening address to 127.0.0.1, since we're not using the RBAC proxy to add any kind of authentication/authorization and someone may be reading the old endpoint.

**Pull request merged**